metadata.json

```diff
@@ -12,15 +12,28 @@
       "operatingsystem": "Debian",
       "operatingsystemrelease": [
         "6",
-        "7"
+        "7",
+        "8"
       ]
     },
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "10.04",
         "12.04",
-        "14.04"
+        "14.04",
+        "15.04",
+        "15.10",
+        "16.04",
+        "16.10",
+        "17.04"
+      ]
+    },
+    {
+      "operatingsystem": "Linux Mint",
+      "operatingsystemrelease": [
+        "13",
+        "17*",
+        "18*"
```

Comment:
are the * here on purpose? Do you know if this is valid syntax? I haven't seen this before.